Pod/Classes/TSSwiftMarkdownParser.swift

```diff
@@ -98,6 +100,19 @@ public class TSSwiftMarkdownParser: TSBaseParser {
         strongAndEmphasisAttributes = [NSFontAttributeName: strongAndEmphasisFont]
         
         if withDefaultParsing {
+            
+            addNumberedListParsingWithMaxLevel(0, leadFormattingBlock: { (attributedString, range, level) in
```

Comment:
Its a starting point.  If there is more levels of indentation then this handles them.  But we start at 0